README.md

```diff
@@ -22,8 +22,6 @@ If you haven't written a post before, please add yourself to `_data/authors.yml`
 
 That's it, we'll take care of the rest. If you wish, you can also submit just a plain Markdown file and we'll be happy to integrate it.
 
-You can also use `tut` in posts. See `posts/2016-09-30-subtype-typeclasses.md` for an example.
```

Comment:
`tut` is archived, and we don't have any mdoc examples. We can investigate that as a follow up though :)